README.md

```diff
@@ -120,6 +120,22 @@ kickstart { '/var/www/html/kickstart.cfg':
 }
 ```
 
+or `packages` parameter with a `Hash` of options and packages:
+
+```
+kickstart { '/var/www/html/kickstart.cfg':
+  packages => {
+    'package_list' => [
```

Comment:
This example might end up being a little confusing since with only the `package_list` key present it's a little redundant. The example you have below (with the `options` key) is sufficient :)